.gitignore

```diff
@@ -28,3 +28,15 @@ temp/
 test/sandbox
 test/.pkgcache/
 .pkgcache/
+intl/MSG.json
```

Comment:
Instead of listing all different languages and files, can we afford to git-ignore everything in `intl` folder? 

BTW, keep in mind that npm excludes git-ignored files from the package by default, so we need extra config to bring `intl` files back when publishing to npmjs.org.